src/index.html

```diff
@@ -6,7 +6,315 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+
+    <div class="page__header">
+      <header class="header">
+        <a href="#" class="logo">
+          <img src="images/logo.png" alt="logo"
+          class="logo__img">
+        </a>
+        <nav class="nav">
```

Comment:
Them BEM metodology here is failing in nesting and membership 
https://en.bem.info/methodology/quick-start/#nesting
I know that it can be counter intuitive sometimes but take your time to improve this topic, it's worth it!